tools/wptserve/wptserve/response.py

```diff
@@ -183,6 +183,9 @@ def iter_content(self, read_file=False):
         True, the entire content of the file will be returned as a string facilitating
         non-streaming operations like template substitution.
         """
+        if not isinstance(self.content, Iterable) and not hasattr(self.content, "read"):
+            raise TypeError("Response content must be an iterable or file-like object!")
```

Comment:
Also, this yields a pretty useless stack trace. Can we turn the response attribute into a getter/setter and check in the setter?